workflow-bot-app/src/test/java/com/symphony/bdk/workflow/versioning/repository/VersionedWorkflowRepositoryTest.java

```diff
@@ -107,21 +107,10 @@ void testFindByWorkflowIdAndActiveTrue() {
 
   @Test
   void findFirstByWorkflowIdOrderByVersionDesc() {
-    Optional<VersionedWorkflow> workflow = versionedWorkflowRepository.findFirstByWorkflowIdOrderByVersionDesc("id1");
+    Optional<VersionedWorkflow> workflow = versionedWorkflowRepository.findByWorkflowIdAndPublishedFalse("id1");
     assertThat(workflow).isPresent();
     assertThat(workflow.get().getWorkflowId()).isEqualTo("id1");
     assertThat(workflow.get().getVersion()).isEqualTo(V3);
   }
 
-  @Test
-  void testDeleteAllByWorkflowId() {
-    boolean exist = versionedWorkflowRepository.existsByWorkflowId("id1");
-    assertThat(exist).isTrue();
-
-    versionedWorkflowRepository.deleteByWorkflowId("id1");
-    List<VersionedWorkflow> all = versionedWorkflowRepository.findAll();
-
-    assertThat(all).hasSize(1);
-    assertThat(all.get(0)).isEqualTo(versionedWorkflow4);
-  }
```

Comment:
why this is removed and not just updated? is the deleteByWorkflowId method still covered somewhere?

Reply:
yes, it is covered by `testDelete_withoutVersion_succeed`